.github/workflows/build_windows_packages.yml

```diff
@@ -16,15 +16,30 @@ on:
 jobs:
   build_windows_packages:
     name: Build Windows Packages
-    runs-on: windows-2022  # TODO(#36): move to cluster of CPU builders like `azure-windows-scale-nod-ai`
+    runs-on: azure-windows-scale-nod-ai
```

Comment:
Let's update the PR title and description to reflect the increased scope here. This PR does much more than "fix ccache" now.

Reply:
Then once this is merged we can close https://github.com/nod-ai/TheRock/pull/80 and delete some pending branches 😄